### DISCUSSION TITLE
ocr识别数字不准，是对字体有要求吗？

### DISCUSSION THREAD

**Discussion by 46928664** [hamibot/hamibot#70] [问题]
好多数字识别不到。比如1，3，7。
还是说我的图分辨率太低？
如果图片分辨率不够，有啥办法？识别命令里好像有个深度的参数，是啥意思？
![pic](https://user-images.githubusercontent.com/115582120/195623463-06561db6-3178-4ef8-bb1f-537b3c8d527e.jpg)

**Comment by batu1579** [comment]
有没有可能你截图以后裁切一下会更容易识别？

**Reply by 46928664** [reply]
这个就是裁切后的图片呢。原图是整个手机桌面，裁切了包含这个数字的最小区域。尝试过扩大一些裁切范围，试过很多范围，都不行。很多数字识别不出来，返回空值，还有的返回是错的，返回一些莫名其妙的字符。

**Comment by batu1579** [comment]
@46928664 
我昨天也试了一下，用内置的 OCR 也没得到结果。我觉得可以考虑试试用大公司们开放的接口，那个可能能识别出来。
另外有没有可能用控件或者别的方式来获取到？毕竟 OCR 的结果不是 100% 确定的，可能会影响脚本的稳定。
